.github/workflows/preDeploy.yml

```diff
@@ -137,3 +137,40 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ github.token }}
           SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK }}
+  newContributorReminder:
```

Comment:
Can we add some empty lines and a comment description between these jobs please?

Reply:
Also, let's call the job `newContributorWelcomeMessage`